lib/buildpackutil.py

```diff
@@ -53,6 +53,18 @@ def get_database_config(development_mode=False):
         'DatabaseHost': match.group('host'),
         'DatabaseName': match.group('dbname'),
     }
+
+    if 'extra' in match.groupdict():
+        extra = match.group('extra').lstrip('?')
+        import urlparse
+        jdbc_params = urlparse.parse_qs(extra)
+        if 'sslmode' in jdbc_params:
+            use_ssl = ['verify-ca', 'verify-full',
```

Comment:
When you set 'DatabaseUseSsl' to true, this implies 2 things. ssl=true and only a TLS connection without certificate validation. This behaves similar to sslmode=require. I think that sslmodes 'verify-ca' and 'verify-full' should not map to this, because they imply verification that is not done. 

Also, if I read the PostgreSQL docs correctluy, allow and prefer throw an exception.

Reply:
So we should only support `require` which results in `DatabaseUseSsl` to `true`. Any other value ignore